Gruntfile.js

```diff
@@ -369,20 +535,32 @@ module.exports = function() {
 		},
 		express: {
 			options: {
-				port: 4002,
+				port: TEST_DEBUG_PORT,
 				hostname: "0.0.0.0"
 			},
 			dev: {
 				options: {
 					script: "tests/server/app.js"
 				}
+			},
+			"secondary": {
+				options: {
+					script: "tests/server/app.js",
+					port: (TEST_DEBUG_PORT + 1)
+				}
+			},
+			doc: {
+				options: {
+					port: 4002,
```

Comment:
@nicjansma Should we replace this with `TEST_DEBUG_PORT - 1`? Just realized that it's still hard coded (duh...)